app/helpers/application_helper/toolbar/ems_storages_center.rb

```diff
@@ -15,18 +15,37 @@ class ApplicationHelper::Toolbar::EmsStoragesCenter < ApplicationHelper::Toolbar
                        :send_checked => true,
                        :confirm      => N_("Refresh relationships and power states for all items related to the selected Storage Managers?"),
                        :enabled      => false,
-                       :onwhen       => "1+"),
+                       :onwhen       => "1+"
+                     ),
                      separator,
+                     button(
+                       :ems_storage_new,
+                       'pficon pficon-add-circle-o fa-lg',
+                       t = N_('Add a New Storage Manager'),
+                       t,
+                       :url => "/new"
+                     ),
+                     button(
+                       :ems_storage_edit,
+                       'pficon pficon-edit fa-lg',
+                       N_('Select a single Storage Manager to edit'),
+                       N_('Edit Selected Storage Manager'),
+                       :url_parms    => "main_div",
+                       :send_checked => true,
+                       :enabled      => false,
+                       :onwhen       => "1"
+                     ),
                      button(
                        :ems_storage_delete,
                        'pficon pficon-delete fa-lg',
                        N_('Remove selected Storage Managers from Inventory'),
-                       N_('Remove Storage Managers from Inventory'),
+                       N_('Remove  Storage Managers from Inventory'),
```

Comment:
Is the extra `' '` here for padding/alignment?

Reply:
No, it is a mistake, will remove.